scanners/android/android.go

```diff
@@ -79,7 +79,7 @@ func (scanner *Scanner) DefaultOptions() models.OptionModel {
 
 	projectLocationOption.AddOption("_", moduleOption)
 	moduleOption.AddOption("_", variantOption)
-	variantOption.AddOption("_", configOption)
+	variantOption.AddConfig("", configOption)
```

Comment:
in the case of `DefaultOptions` should not `_` remain?

Reply:
I think we shall not force it to be filled as it is not a required field.